nuxt.config.ts

```diff
@@ -88,6 +88,10 @@ const config: Configuration = {
     {
       src: '@/plugins/vue-chart.ts',
       ssr: true
+    },
+    {
+      src: '@/plugins/vuetify.js',
```

Comment:
```suggestion
      src: '@/plugins/vuetify.ts',
```
ここ `.ts` である必要があります 🙇